bin/start_cn.sh

```diff
@@ -0,0 +1 @@
+start_be.sh
```

Comment:
why not copy from `start_be.sh `and change it to start_cn.
I think it will make things more simple.
so do the stop_cn.sh